metron-stellar/stellar-common/src/main/java/org/apache/metron/stellar/common/utils/VFSClassloaderUtil.java

```diff
@@ -112,14 +112,18 @@ public static FileSystemManager generateVfs() throws FileSystemException {
    * @throws FileSystemException
    */
   public static Optional<ClassLoader> configureClassloader(String paths) throws FileSystemException {
+    LOG.debug("Configuring class loader with paths = {}", paths);
     if(paths.trim().isEmpty()) {
```

Comment:
Should we add a null check here, just in case?  Or use StringUtils.isBlank?

Reply:
We haven't encountered any instances with path being null that I'm aware of - I'd like to start with this and add more checks as needed.